service/apiserver/server.api

```diff
@@ -30,6 +30,12 @@ type ReqGetRange {
 	Limit  uint32 `form:"limit,range=[1:100]"`
 }
 
+type ReqGetRangeWithStartId {
+	Offset  uint32 `form:"offset,range=[0:100000]"`
+	Limit   uint32 `form:"limit,range=[1:100]"`
+	StartId int64  `form:"start_id"`
```

Comment:
The name `StartId` seems not very good. Any other ideas? `from` or `fromId` or ...

Reply:
`fromId` looks good